ui/lib/core/addon/components/info-table-row.js

```diff
@@ -19,7 +19,11 @@ import layout from '../templates/components/info-table-row';
  * @param helperText=null {string} - Text to describe the value displayed beneath the label.
  * @param alwaysRender=false {Boolean} - Indicates if the component content should be always be rendered.  When false, the value of `value` will be used to determine if the component should render.
  * @param [type=array] {string} - The type of value being passed in.  This is used for when you want to trim an array.  For example, if you have an array value that can equal length 15+ this will trim to show 5 and count how many more are there
+ * @param [isLink=true] {Boolean} - Passed through to InfoTableItemArray. Indicates if the item should contain a link-to component.  Only setup for arrays, but this could be changed if needed.
```

Comment:
Though these properties are really for the dependent `info-table-item-array` they are passed through in this component first.  Not sure our thoughts on adding documentation for an essentially nested component in the parent.  My thought is it's helpful, so I added it to this component as well.  Open to feedback.

Reply:
For the other two specifically this makes sense. My only question is, if we're showing an array (which is a new type on info-table) can we assume isLink is true for now? And if we run into a future case where it's not, we abstract it out in this way?

Reply:
Arrays are actually used elsewhere in the app.  The originally if array conditional on info-table-row was there before I changed things in there (See row 32 of `info-table-row`).  As such, I made isLink false by default so I don't impact these other array types.